src/script/lua_api/l_util.cpp

```diff
@@ -329,6 +329,51 @@ int ModApiUtil::l_mkdir(lua_State *L)
 	return 1;
 }
 
+// rmdir(path, recursive)
+int ModApiUtil::l_rmdir(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	const char *path = luaL_checkstring(L, 1);
+	CHECK_SECURE_PATH(L, path, true);
+
+	bool recursive = false;
+
+	if (!lua_isnil(L, 2)) {
+		recursive = lua_toboolean(L, 2);
+	}
+
+	if (recursive) {
+		lua_pushboolean(L, fs::RecursiveDelete(path));
+	} else {
+		lua_pushboolean(L, fs::DeleteSingleFileOrEmptyDirectory(path));
+	}
```

Comment:
For example this can be:
```
	if (recursive)
		lua_pushboolean(L, fs::RecursiveDelete(path));
	else
		lua_pushboolean(L, fs::DeleteSingleFileOrEmptyDirectory(path));
```